### ISSUE TITLE
Error on load UE 5.2 Linux

### ISSUE THREAD

**Issue by ramAdam** [ramAdam/PaperZD#1]
After installation it runs fine , when you start the project again, I get this error !!
![Screenshot from 2023-10-08 16-24-54](https://github.com/ramAdam/PaperZD/assets/4009156/57a6e454-8a45-4092-bcfe-00a18be8c5ec)

**Comment by ramAdam** [comment]
Cleaning the binaries before starting the engine is my work around for now!

**Comment by arturmattner** [comment]
Unfortunately just deleting the Binaries folder wasn't enough in my case, I had to delete the DerivedDataCache, Intermediate, and Saved folders as well in a new project. Once I did that I rebuild the project and it worked. Thank you for providing the script! I used it against PaperZD for UE5.4 and it worked without issue.

**Comment by ramAdam** [comment]
@arturmattner that's right, i had to write another script later on to clean up the folders that you mentioned in your comment.

**Comment by ssrhunter** [comment]
> Unfortunately just deleting the Binaries folder wasn't enough in my case, I had to delete the DerivedDataCache, Intermediate, and Saved folders as well in a new project. Once I did that I rebuild the project and it worked. Thank you for providing the script! I used it against PaperZD for UE5.4 and it worked without issue.

I'm getting the same error that is shown in the above screenshot.

If I delete the folders that you mentioned, I get a message about the project needing to be built from source. I'm not sure how to proceed from that point, because it gives an error and says the project can't be rebuilt. Could you give some more information on your build process?

**Comment by ramAdam** [comment]
@ssrhunter have you solved the problem ? here is the script i was using to clean the folder and build the game from terminal

`

> #!/bin/bash
> 
> folder_path="Binaries/Linux"
> 
> clean_folder() {
>     if [ -d "$folder_path" ]; then
>         if [ -z "$(ls -A $folder_path)" ]; then
>             echo "Folder is already empty. Nothing to clean."
>         else
>             rm -r -v "$folder_path"/*   # -r for recursive, -v for verbose
>             echo "Folder cleaned successfully."
>         fi
>     else
>         echo "Error: Folder not found."
>     fi
> }
> 
> build_editor() {
>     print_highlighted "Building NameOfYourGame..."  # <----- replace it with the name of your game
>     make NameOfYourGame # <---- replace it with the name of your game
>     if [ $? -eq 0 ]; then
>         print_highlighted "Build successful!" # Highlighted in bold and green
>     else
>         print_highlighted "Build failed. Please check the error messages above." # Highlighted in bold and red
>     fi
> }
> 
> 
> read -p "Do you want to clean the folder $(print_highlighted "$folder_path")? (y/n): " clean_choice
> read -p "Do you want to build SunSetRiders? (y/n): " build_choice
> 
> if [[ "$clean_choice" =~ [Yy] ]]; then
>     echo "Cleaning folder: $(print_highlighted "$folder_path")"
>     clean_folder
> fi
> 
> if [[ "$build_choice" =~ [Yy] ]]; then
>     build_editor
> fi

`


> If I delete the folders that you mentioned, I get a message about the project needing to be built from source. I'm not sure how to proceed from that point, because it gives an error and says the project can't be rebuilt. Could you give some more information on your build process?

folder_path is "Binaries/Linux" in  your project directory, when the engine gives an error,  try to clean the binaries folder again and proceed to rebuild, for me cleaning the binaries in the folder path mentioned above, does the trick.

be extra careful with the folder path if you are using the shell script, you can put this shell script in the root folder of your project. Clean the binaries with this script before starting your project and then let the engine build the binaries for you. if the engine gives you an error, clean and rebuild again !

You can also try to rebuild from the game from the script, replace "NameOfYourGame" with the name of your project name.

**Comment by ssrhunter** [comment]
Thanks @ramAdam . Yes, I was able to resolve the issue. I've been using a different workflow where I build PaperZD from source using a custom shell script.
